wiki/Configuration:-Input.md

```diff
@@ -132,6 +141,7 @@ A few settings are common between `touchpad`, `mouse` and `trackpoint`:
 
 Settings specific to `touchpad`s:
 
+- `disabled-on-external-mouse`: do not send events while external pointer device is plugged in.
```

Comment:
Could you move this to the bottom of this list?

Reply:
Sure. Updated struct field, too.